build/images/kubectl-delivery/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM alpine:3.17 AS build
+
+# Install kubectl.
+ENV K8S_VERSION v1.27.0
```

```suggestion
ENV K8S_VERSION v1.27.5
```

Let's use the latest patch version.

---

Agreed